src/js/components/SearchBar/Input.tsx

```diff
@@ -25,10 +25,11 @@ const StyledInput = styled.input`
     outline: none;
   }
 `
+// create a tiny little history closure
```

Comment:
Is this comment referring to this component or just a leftover note to self?

Reply:
A left over note to self!